jax/_src/lax/lax.py

```diff
@@ -551,9 +557,10 @@ def conv_general_dilated(
     feature_group_count: integer, default 1. See XLA HLO docs.
     batch_group_count: integer, default 1. See XLA HLO docs.
     precision: Optional. Either ``None``, which means the default precision for
-      the backend, a ``lax.Precision`` enum value (``Precision.DEFAULT``,
-      ``Precision.HIGH`` or ``Precision.HIGHEST``) or a tuple of two
-      ``lax.Precision`` enums indicating precision of ``lhs``` and ``rhs``.
+      the backend, a string ('bfloat16', 'tensorfloat32', or 'float32'), a
+      ``lax.Precision`` enum value (``Precision.DEFAULT``, ``Precision.HIGH`` or
+      ``Precision.HIGHEST``) or a tuple of two ``lax.Precision`` enums
+      indicating precision of ``lhs``` and ``rhs``.
```

Comment:
Could we also support a tuple of two strings for this case? E.g., `precision=('bfloat16', 'float32')` would be nice than requiring a pair of enums.